src/shared/logger/consoleLogTransport.ts

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as Transport from 'winston-transport'
+
+interface LogEntry {
+    level: string
+    message: string
+}
+
+export class ConsoleLogTransport extends Transport {
```

Comment:
What does this class do? maybe add a comment about it?

Reply:
It is a simple log-to-console.log for winston. I've added a comment